scvi/module/_jaxvae.py

```diff
@@ -76,15 +76,15 @@ def setup(self):
         self.dense4 = Dense(self.n_hidden)
         self.dense5 = Dense(self.n_input)
 
-        training = not self.is_training
+        is_eval = not self.is_training
         self.batchnorm1 = nn.BatchNorm(
-            momentum=0.99, epsilon=0.001, use_running_average=training
+            momentum=0.99, epsilon=0.001, use_running_average=is_eval
         )
         self.batchnorm2 = nn.BatchNorm(
-            momentum=0.99, epsilon=0.001, use_running_average=training
+            momentum=0.99, epsilon=0.001, use_running_average=is_eval
         )
-        self.dropout1 = nn.Dropout(self.dropout_rate, deterministic=training)
-        self.dropout2 = nn.Dropout(self.dropout_rate, deterministic=training)
+        self.dropout1 = nn.Dropout(self.dropout_rate, deterministic=is_eval)
```

Comment:
Do we need dropout for the decoder if we never use it?

Reply:
Since its just a generalized decoder component I think we can leave it. We just dont happen to use it in JaxVAE specifically